app/components/ui/atoms/NotificationRibbon.js

```diff
@@ -0,0 +1,24 @@
+import styled from 'styled-components'
+import { th } from '@pubsweet/ui-toolkit'
+
+const backgroundColor = ({ type = 'primary', theme }) =>
+  ({
+    error: theme.colorError,
+    success: theme.colorSuccess,
+    primary: theme.colorPrimary,
+  }[type])
```

Comment:
We now have this helper function in three places. Surely it's time to factor it out into a common place?

Reply:
While this takes type as input (as opposed to validationStatus), I agree, there should be a function like `getColor` somewhere in pubsweet that accounts for every possibility (error, success, warning, primary etc).